index.html

```diff
@@ -1020,6 +1119,70 @@ <h2>
         </dd>
       </dl>
     </section>
+    <section data-dfn-for="GamepadTouch">
+      <h2>
+        <dfn>GamepadTouch</dfn> Interface
+      </h2>
+      <p>
+        This interface defines a single touch event on a gamepad device that
+        supports input. The event consists of a touch id that uniquely
+        identifies the touch point from the time the input medium (e.g. finger,
+        stylus, etc) makes contact with the touch device, up to the time the
+        input medium is no longer making contact with the touch device.
+      </p>
+      <pre class="idl" data-cite="geometry-1">
+        dictionary GamepadTouch {
+          unsigned long touchId;
+          octet surfaceId;
+          DOMPointReadOnly position;
+          DOMRectReadOnly? surfaceDimensions;
+        };
+      </pre>
+      <dl>
+        <dt>
+          <dfn>touchId</dfn> attribute
+        </dt>
+        <dd>
+          Unique id of the touch event. Range is [0, 4294967295].
+          <p>
+            The user agent is responsible for:
+          </p>
+          <ul>
+            <li>Incrementing the id for each subsequent touch event based on
+            information provided by the device API.
+            </li>
+            <li>Maintaining the uniqueness of the id per origin to prevent
```

Comment:
I think uniqueness is not the right word since we expect all the ID values to be initialized to the same value, also "per origin" is not right since we expect two tabs from the same origin to not use the same touch IDs.

Maybe "Maintaining per-tab touch ids to prevent fingerprinting" ?

Reply:
(or just remove this section completely as we discussed)